model/uzivatel.php

```diff
@@ -256,11 +256,31 @@ function historiePrihlaseni() {
   }
 
   /** Jméno a příjmení uživatele v běžném (zákonném) tvaru */
-  function jmeno()
-  {
+  function jmeno() {
     return $this->u['jmeno_uzivatele'].' '.$this->u['prijmeni_uzivatele'];
   }
 
+  /** Vrátí adresu uživatele ve formátu Město, Ulice ČP, PSČ, stát*/
+  function adresa() {
+    $adresa = $this->u['mesto_uzivatele'].', '.$this->u['ulice_a_cp_uzivatele'].', '.$this->u['psc_uzivatele'].', '.$this->stat();
+    return $adresa;
+  }
+
+  /** Vrátí číslo občanského průkazu uživatele */
+  function obcanskyPrukaz() {
+    return $this->u['op'];
+  }
+
+  /** Vrátí datum narození uživatele jako string ve formátu dd.mm.rrrr nebo prázdný řetězec*/
+  function datumNarozeniUzivatele() {
```

Comment:
Už jsem zjistil jak se to řeší, tady by se měl vrátit objekt (resp. objekt nebo null, pokud má být null legální hodnota) a potom v šabloně mít něco typu {uzivatel.datumNarozeni.formatDatumStandard}.

Mělo by se to jmenovat jen datumNarozeni, že se týká uživatele je již dané tím, že je to v tříde Uzivatel.

Pokud zatím nechceme slučovat uživatele, kteří datum nemají vyplněné, pak je korektní chování, že v šabloně v případě zobrazení uživatele bez data narození vznikne výjimka (tj. že nebudeš situaci, že to vrací null nijak ošetřovat). Pravděpodobně to zjednoduší kód (jak jsem popsal konstrukci pro šablonu výš), což je převažující výhoda. Kdyby se výjimky objevily, uvidíme je v logu a odladíme následně, pokud to bude dávat smysl.